flood_adapt/api/output.py

```diff
@@ -36,8 +38,18 @@ def get_aggregation(name: str, database: IDatabase):
 
 
 def make_infographic(name: str, database: IDatabase) -> str:
-    return database.get_scenario(name).infographic()
+    # Get the scenario
+    scenario = database.get_scenario(name)
+
+    # Check if the scenario has run
+    scenario.has_run_impact = (
```

Comment:
This is not checking, but an assignment?
Is it what you intended to do?

Reply:
It should later be a check but from what I understood, this isn't implemented yet so that is why it is now an assignment. Is this implemented now in the FIAT adapter?

Reply:
Ohw ok, I see.
I think it already happens in floodadapt: https://github.com/Deltares/FloodAdapt/blob/main/flood_adapt/object_model/direct_impacts.py#L89C9-L89C19

Reply:
Als het goed is zo gefixt